app/src/main/java/com/example/lhi06/residue/AlbumAdapter.java

```diff
@@ -18,6 +18,7 @@
 
     private final Picasso picasso;
     private List<Album> albumList = new ArrayList<>();
```

Comment:
I prefer to use plural instead of list in the name of lists so I would suggest changing this to `albums`